modules/stanford_profile_styles/css/stanford_page.css

```diff
@@ -5,3 +5,14 @@
   max-width: 980px;
   margin: auto auto;
 }
+
+@media only screen and (max-width: 992px) {
+  .basic-page.node--layout-full .layout-pacman--left > div > aside {
```

Comment:
we can remove these since we're removing pacman from the layouts

Reply:
Yeah, I'll have to revise now.